config/routes.rb

```diff
@@ -7,8 +7,9 @@
       get '/finished_books' => 'list_items#finished_books'
       post '/list_item' => 'list_items#show'
       post '/add_list_item' => 'list_items#create'
-      # post '/add_list_item' => 'list_items#add_list_item'
-      post '/edit_list_item' => 'list_items#update'
+      get '/list_item/:id/edit' => 'list_items#edit'
```

Comment:
FYI you can also make use of `resources :list_item` - https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default